packages/pyxis-react/src/internalComponents/CardGroup/Control.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, InputHTMLAttributes } from 'react';
+import classNames from 'classnames';
+import { IconProps } from 'components/Icon';
+import { Size, Type } from './CardGroup';
+
+const getClasses = (
+  isLarge:boolean,
+  error:boolean,
+  isChecked?: boolean,
+  disabled?:boolean,
+  className?: string,
+):string => classNames(
+  'form-card',
+  {
+    'form-card--large': isLarge,
+    'form-card--checked': isChecked,
+    'form-card--error': error,
+    'form-card--disabled': disabled,
+  },
+  className,
+);
+
+const getAddonClasses = (isIconAddon:boolean) => classNames(
+  'form-card__addon',
+  { 'form-card__addon--with-icon': isIconAddon },
+);
+
+const Control: FC<ControlProps> = ({
+  additionalText,
+  addon,
+  checked,
+  className,
+  disabled,
+  hasError = false,
+  id,
+  size = 'medium',
+  text,
+  title,
+  type,
+  ...restProps
+}) => {
+  const isLarge = size === 'large';
+  const isIconAddon = !!(addon && typeof addon !== 'string');
+  return (
+    <label
+      className={getClasses(isLarge, hasError, checked, disabled, className)}
+      data-testid={id ? `${id}-wrapper` : undefined}
+    >
+      {addon && (
+        <span className={getAddonClasses(isIconAddon)}>
+          {isIconAddon ? addon({}) : <img alt="" height={70} src={addon} width={70} />}
```

Comment:
Passiamolo l'alt, meglio che lasciarlo sempre vuoto

```suggestion
          {isIconAddon ? addon({}) : <img alt={restProps.alt} height={70} src={addon} width={70} />}
```

Reply:
Essendo un'immagine decorativa, l'alt può essere vuoto